themes/mbriggs/mbriggs.theme.bash

```diff
@@ -1,28 +1,33 @@
+# shellcheck shell=bash
+# shellcheck disable=SC2034
 # ------------------------------------------------------------------#
 #          FILE: mbriggs.zsh-theme                                  #
 #            BY: Matt Briggs (matt@mattbriggs.net)                  #
 #      BASED ON: smt by Stephen Tudor (stephen@tudorstudio.com)     #
 # ------------------------------------------------------------------#
 
-SCM_THEME_PROMPT_DIRTY="${red}⚡${reset_color}"
-SCM_THEME_PROMPT_AHEAD="${red}!${reset_color}"
-SCM_THEME_PROMPT_CLEAN="${green}✓${reset_color}"
+SCM_THEME_PROMPT_DIRTY="${red?}⚡${reset_color?}"
+SCM_THEME_PROMPT_AHEAD="${red?}!${reset_color?}"
+SCM_THEME_PROMPT_CLEAN="${green?}✓${reset_color?}"
 SCM_THEME_PROMPT_PREFIX=" "
 SCM_THEME_PROMPT_SUFFIX=""
-GIT_SHA_PREFIX=" ${yellow}"
-GIT_SHA_SUFFIX="${reset_color}"
+GIT_SHA_PREFIX=" ${yellow?}"
+GIT_SHA_SUFFIX="${reset_color?}"
 
 function git_short_sha() {
 	SHA=$(git rev-parse --short HEAD 2> /dev/null) && echo "$GIT_SHA_PREFIX$SHA$GIT_SHA_SUFFIX"
 }
 
 function prompt() {
 	local return_status=""
-	local ruby="${red}$(ruby_version_prompt)${reset_color}"
-	local user_host="${green}\h${reset_color}"
+	local ruby
+	ruby="${red?}$(ruby_version_prompt)${reset_color?}"
+	local user_host
+	user_host="${green?}\h${reset_color?}"
```

Comment:
Is this needed? When the right-hand side contains command substitutions `$(...)`, Shellcheck will complain that the exit status of the command is masked. However, this line doesn't contain any command substitutions.

Reply:
but shellcheck did complain. you prefer I disable a few more checks?

Reply:
What kind of error does Shellcheck produce?

I tried it in my environments, but Shellcheck doesn't complain about this.

```console
$ shellcheck --version
ShellCheck - shell script analysis tool
version: 0.10.0
license: GNU General Public License, version 3
website: https://www.shellcheck.net
```

In another host,

```console
$ shellcheck --version
ShellCheck - shell script analysis tool
version: 0.11.0
license: GNU General Public License, version 3
website: https://www.shellcheck.net
```